components/Collections/src/MDCCollectionViewController.h

```diff
@@ -49,9 +49,6 @@
  as well as ink during the highlight/unhighlight states.
  */
 
-- (BOOL)collectionView:(nonnull UICollectionView *)collectionView
```

Comment:
This is in the public API.  If we want to remove it, we should go through the deprecation process.  I could see people subclassing MDCCollectionViewController just to overwrite methods like this.